platforms/shared/tests/test-client/main.cpp

```diff
@@ -265,8 +266,8 @@ int main(int argc, char** argv)
     LOG(INFO) << "Instrument ID: " << instrumentId;
 
     testUploadTracks(client, accountId, instrumentId);
+    testUploadCount(client, accountId, instrumentId);
```

Comment:
I'd move testUploadTracks() below the return, so it is still in code, but isn't called